op-challenger/game/fault/contracts/oracle.go

```diff
@@ -6,12 +6,18 @@ import (
 
 	"github.com/ethereum-optimism/optimism/op-bindings/bindings"
 	"github.com/ethereum-optimism/optimism/op-challenger/game/fault/types"
+	"github.com/ethereum-optimism/optimism/op-challenger/game/keccak/matrix"
 	"github.com/ethereum-optimism/optimism/op-service/sources/batching"
 	"github.com/ethereum-optimism/optimism/op-service/txmgr"
 	"github.com/ethereum/go-ethereum/common"
 )
 
 const (
+	methodInitLPP      = "initLPP"
+	methodAddLeavesLPP = "addLeavesLPP"
+	// TODO(client-pod#480): dispute invalid large preimages using challenge LPP
+	// methodChallengeLPP              = "challengeLPP"
```

Comment:
nit: Just delete this - it's trivial content so no need to keep it around and may as well keep things tidy.